acceptance/Rakefile

```diff
@@ -18,16 +18,11 @@ module HarnessOptions
     :type => 'aio',
     :helper => ['lib/helper.rb'],
     :tests  => ['tests'],
-    :log_level => 'info',
-    :color => false,
-    :root_keys => true,
     :ssh => {
       :keys => ["~/.ssh/id_rsa-acceptance"],
     },
-    :xml => true,
     :timesync => false,
```

Comment:
Would enabling this make beaker call timesync on all hosts? If so (and *only* if QENG-3786 is fixed), we should set this to true then delete setup/common/005_SyncTime.rb